powqutyd/files/src/event_handling.c

```diff
@@ -3,16 +3,22 @@
  *  Created on Jun 27, 2017
  *  	Author: ikstream
  */
+#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
+#include <time.h>
 
 #include "event_handling.h"
 #include "libwebslack.h"
 
-#define MAX_MSG_LENGTH 1024
-#define MAX_EVENT_LENGTH 64
+#define MAX_MSG_LENGTH		1024
+#define MAX_EVENT_LENGTH	64
+#define MAX_TIME_LENGTH		64
```

Comment:
only in the webinterface. If you apply them thez are all on the same level. at least in my vim
![snip_2017-07-12-14 14 57](https://user-images.githubusercontent.com/5787845/28117161-90ff79bc-670c-11e7-8d31-490a2e5c4ad7.jpg)